physionet-django/console/templates/console/console_navbar.html

```diff
@@ -167,6 +167,11 @@
                   <a id="code_of_conduct_nav" class="nav-link" href="{% url 'code_of_conduct_list' %}">Code of Conduct</a>
                 </li>
               {% endif %}
+              {% if perms.events.add_event %}
+                <li class="nav-item {% if event_agreement_nav %}active{% endif %}">
+                  <a id="event_agreement_nav" class="nav-link" href="{% url 'event_agreement_list' %}">Event Agreements</a>
```

Comment:
For consistency - rebase on dev to get the other changes to nav

```suggestion
                  <a id="nav_event_agreement" class="nav-link" href="{% url 'event_agreement_list' %}">Event Agreements</a>
```